### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/request-files.md`

### PR DESCRIPTION
**Pull request by lasr3**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]
Great, 🚀.

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great @devluisrodrigues ✨

File: docs/pt/docs/tutorial/request-files.md [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+
+Isso é necessário, visto que os arquivos enviados são enviados como "dados de formulário".
+
+///
+
+## Importe `File`
+
+Importe `File` e `UploadFile` de `fastapi`:
+
+//// tab | Python 3.9+
```
[the remark this answers is not in the document]
Comment (devluisrodrigues) [reply docs/pt/docs/tutorial/request-files.md:23]: Ok, I'll take a look, and commit the changes. Thank you

**Comment by devluisrodrigues** [comment]
Thank you for your feedback @ceb10n and @nazarepiedady . I'll adress the changes and update my PR

**Comment by devluisrodrigues** [comment]
> 📝 Docs preview for commit [cb7cbdb](https://github.com/fastapi/fastapi/commit/cb7cbdb6443880fd7f485a81fb1a2015d7c6ba83) at: https://36fb23ee.fastapitiangolo.pages.dev
> ### Modified Pages
> 
>     * https://36fb23ee.fastapitiangolo.pages.dev/pt/tutorial/request-files/

Oh perfect, I can see now. Much cleaner interface!
Thank you @ceb10n !!

File: docs/pt/docs/tutorial/request-files.md [hunk 2]
```diff
@@ -221,91 +137,13 @@ Isso não é uma limitação do **FastAPI**, é parte do protocolo HTTP.
...
-Prefira usar a versão `Annotated` se possível.
-
-///
-
-```Python hl_lines="9  17"
-{!> ../../docs_src/request_files/tutorial001_02.py!}
-```
-
-////
+{* ../../docs_src/request_files/tutorial001_02_an_py310.py hl[17] *}
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/request-files.md:140]: missed the highlight of the line 9:

{* ../../docs_src/request_files/tutorial001_02_an_py310.py hl[9,17] *}

File: docs/pt/docs/tutorial/request-files.md [hunk 3]
```diff
@@ -221,91 +137,13 @@ Isso não é uma limitação do **FastAPI**, é parte do protocolo HTTP.
...
-Prefira utilizar a versão `Annotated` se possível.
-
-///
-
-```Python hl_lines="7  13"
-{!> ../../docs_src/request_files/tutorial001_03.py!}
-```
-
-////
+{* ../../docs_src/request_files/tutorial001_03_an_py39.py hl[15] *}
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/request-files.md:146]: here to:

{* ../../docs_src/request_files/tutorial001_03_an_py39.py hl[9,15] *}

File: docs/pt/docs/tutorial/request-files.md [hunk 4]
```diff
@@ -315,49 +153,7 @@ Eles serão associados ao mesmo "campo de formulário" enviado usando "dados de
...
-Prefira utilizar a versão `Annotated` se possível.
-
-///
-
-```Python hl_lines="10  15"
-{!> ../../docs_src/request_files/tutorial002.py!}
-```
-
-////
+{* ../../docs_src/request_files/tutorial002_an_py39.py hl[15] *}
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/request-files.md:156]: here to:

{* ../../docs_src/request_files/tutorial002_an_py39.py hl[10,15] *}

**Review by ceb10n — CHANGES_REQUESTED** [review]
Thank you ❤️ 

Just missing some highlights

File: docs/pt/docs/tutorial/request-files.md (hunk 4 above)
Comment (devluisrodrigues) [reply docs/pt/docs/tutorial/request-files.md]: Ohh sorry, I got a little confused. I just fixed it!

**Comment by devluisrodrigues** [comment]
Thank you again @ceb10n! Just fixed the missed highlights!

**Review by ceb10n — APPROVED** [review]
🚀

**Comment by alejsdev** [comment]
Thanks for your help! @devluisrodrigues :partying_face: 
And thanks for the detailed review @ceb10n :100:

**Pull request merged**